CMakeLists.txt

```diff
@@ -143,6 +143,8 @@ else()
   add_subdirectory(OrbitLinuxTracing)
   add_subdirectory(OrbitService)
   add_subdirectory(OrbitClientGgp)
+  add_subdirectory(OrbitCaptureGgpClient)
```

Comment:
I think we had discussed this before, not sure what the conclusion was: Does this actually have anything to do with ggp, or is this just a generic Linux client? (The name seems to suggest there is something special to ggp about this.)

No need to change anything now, but we should clarify.

Reply:
It should work in Linux as well but it is meant to be used in ggp that is why I called it like that. Happy to change it to a better and more clear name. I think the names need a review anyway as they can be quite confusing (Daniel mentioned it as well offline) so we can have a meeting to discuss about them, agree which should be keep and which should be updated and then I will create a PR for that.